src/sys/unix.rs

```diff
@@ -482,6 +482,16 @@ impl SockAddr {
 
 pub(crate) type Socket = c_int;
 
+pub(crate) fn socket_from_raw(socket: Socket) -> crate::socket::Inner {
+    unsafe { crate::socket::Inner::from_raw_fd(socket) }
+}
```

Comment:
Nit: empty lines after function bodys. Doesn't rustfmt enforce this?

Reply:
@Thomasdezeeuw There is an unstable Rustfmt option [`blank_lines_lower_bound`](https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#blank_lines_lower_bound) which enforces this, you might be interested in enabling it.

Reply:
Doesn't seem to work right just quite yet: https://github.com/rust-lang/rustfmt/issues/3382#issuecomment-825168920, perhaps once it becomes stable.